File: python/Stack/stack/Uimanager.py
from .Book import Book
from .Shelf import Shelf
from .filemanager import Filemanager

class Uimanager:
    def menu(self):
        print("***** Menu *****")
        print("1.add book to shelf")
        print("2.remove book from shelf ")
        print("3.display books ")
        print("4. save data to file ")
        print("5.Exit")

    def getchoice(self):
        choice= int(input("enter your choice :  "))
        return choice 
   


    

File: c++/LinkedList/utils/uimanager.cpp
#include"filemanager.cpp"
#include <iostream>
using namespace std ;
#ifndef UTILS_UIMANAGER_CPP
#define UTILS_UIMANAGER_CPP
class UIManager
{
    public:

        void showMenu(){
            cout<<"***** Menu *****"<<endl ;
            cout <<"1. insert new node "<<endl ;
            cout <<"2.search node "<<endl ;
            cout <<"3.display all nodes  "<<endl ;
            cout <<"4. remove node "<<endl ;
            cout<<"5.save and exit "<<endl ;
        
        }
        int getChoice (int choice ){
            cout <<"enter your choice "<< endl ; 
            cin >> choice;
            return choice;
        }
};
#endif // UTILS_UIMANAGER_CPP